acceptancetests/assess_persistent_storage.py

```diff
@@ -120,29 +110,120 @@ def assess_charm_deploy_and_remove_single_storage(client):
                 log.error(
                     'Name mismatch on Single block device storage, test terminated.')
                 sys.exit(1)
+            else:
+                log.info('Check name and total number of storage unit: PASSED.')
 
-        # check type, persistent setting and volume of single block storage unit
+        # check type, persistent setting and pool of single block storage unit
         storage_type = storage_output['storage'][single_blk_id]['kind']
         persistent_setting = storage_output['storage'][single_blk_id]['persistent']
         pool_storage = storage_output['volumes']['0']['storage']
         pool_setting = storage_output['volumes']['0']['pool']
+
         if storage_type != 'block':
             log.error(
-                'Incorrect type for single block device storage detected \
-                - {}, test terminated.'.format(storage_type))
+                'Incorrect type for single block device storage detected '\
+                '- {}, test terminated.'.format(storage_type))
             sys.exit(1)
-        elif persistent_setting != 'true':
+        # Be careful, the type of persistent_setting's value is boolean.
+        # Using != to clearly highlight it here.
+        elif persistent_setting != True:
```

Comment:
These should be assertions, not if statements with exits and error logging. Can you clean this up to make it more readable, as well as methodized so you aren't repeating yourself. Setting up a dict of expected strings would make this comparison easy and short.